### PR TITLE
Copying complete vendor repo of SVTTestRepo

### PR DESCRIPTION
**Pull request by psoujany**
Copying complete vendor repo of SVTTestRepo to openjdk-tests/system folder, which can be used to perform SVTTestRepo conversion of z/OS.

This fixes
runtimes/backlog/issues/517

### CODE REVIEW

**Comment by psoujany** [comment]
Below is the verification job where copy happened successfully.
hyc-runtimes-jenkins/view/Test_grinder/job/Grinder/15735

File: get.sh [hunk 1]
```diff
@@ -555,7 +555,10 @@ getFunctionalTestMaterial()
 			if [[ "$dir" != "" ]] && [[ -d $dest/$dir ]]; then
 				echo "Stage $dest/$dir to $TESTDIR/$dir"
 				# already in TESTDIR, thus copy $dir to current directory
-				cp -r $dest/$dir ./
+					cp -r $dest/$dir ./
```
Comment (Mesbah-Alam) [inline get.sh:558]: Please undo this change in indentation at line 558. It's not required.

**Comment by Mesbah-Alam** [comment]
This PR is related to SVTTestRepo/pull/30.

File: get.sh (hunk 1 above)
Comment (psoujany) [reply get.sh]: Modified could you please check now.

File: get.sh [hunk 2]
```diff
@@ -556,6 +556,9 @@ getFunctionalTestMaterial()
 				echo "Stage $dest/$dir to $TESTDIR/$dir"
 				# already in TESTDIR, thus copy $dir to current directory
 				cp -r $dest/$dir ./
+				if [[ "$PLATFORM" == "s390x_zos" ]]; then
```
Comment (llxia) [inline get.sh:559]: We also need to consider 31bit zos. Please check if $PLATFORM contains `zos`

Comment (psoujany) [reply get.sh]: Modified the check to address 31/64 bit z/OS. 
Verification job 
hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/15741

**Review by Mesbah-Alam — APPROVED** [review]
LGTM

**Comment by Mesbah-Alam** [comment]
Approving based on test result at svt-jenkins/Grinder/15742

**Review by llxia — APPROVED** [review]
LGTM
